extensions/schemas/standard/clause_0_front_material.adoc

```diff
@@ -1,37 +1,37 @@
+To use a schema for data validation, the schema must be converted into a schema representation suitable for validating data in the specific data format. For example, an XML Schema that is a GML application schema or a JSON Schema for GeoJSON or the draft OGC Features and Geometries JSON (JSON-FG).
 
-While the schema is a logical schema, a machine-readable representation is needed. Instead of specifying a custom schema language for geospatial data, this Standard uses JSON Schema as the schema language with 
+While the schema is a logical schema, a machine-readable representation is needed. Instead of specifying a custom schema language for geospatial data, this Standard uses JSON Schema as the schema language with: 
 
-* extensions for additional aspects that are common in geospatial data; and 
-* recommendations for restricting the use of JSON Schema capabilities in order to simplify parsing a schema.
+* Extensions for additional aspects that are common in geospatial data; and 
+* Recommendations for restricting the use of JSON Schema capabilities in order to simplify parsing a schema.
 
 That is, JSON Schema is primarily used as a data description language.
 
 The main reasons for using JSON Schema are:
 
 * Web developers are often familiar with JSON Schema and JSON;
-* JSON data types (string, number, boolean, array, object, null) are simple and easy to understand; and
+* JSON data types (string, number, Boolean, array, object, null) are simple and easy to understand; and
 * JSON is often used for encoding geospatial data, i.e., the schema of a property can be used to validate property values without conversion.
 
-This Standard first specifies <<rc_schemas,basic provisions for schemas that are applicable to any type of geospatial data>>. These provisions are not specific to features and as such could be moved to OGC API Common in the future.
+This Standard first specifies <<rc_schemas,basic provisions for schemas that are applicable to any type of geospatial data>>. These provisions are not specific to features and as such could be moved to OGC API - Common in the future.
 
-Additional provisions are added for <<rc_core-roles-features,schemas for feature data>> as well as <<rc_feature-references,provisons for references to other features>>.
+Additional provisions are added for <<rc_core-roles-features,schemas for feature data>> as well as <<rc_feature-references,provisions for references to other features>>.
 
-In OGC Web APIs, geospatial data is shared in collections (path `/collections/{collectionId}`). The schema of items in a collection provides information how to interact with the collection. 
+In the OGC Web API Standards suite, geospatial data is shared in collections (path `/collections/{collectionId}`). The schema of items in a collection provides information as to how to interact with the collection. 
```

Comment:
"OGC Web APIs" is correct here, this is a reference to APIs, not the standards.

Reply:
Meeting 2024-05-06: Keep all uses of "OGC Web API". The definition of the term is from Part 1 and also in this document.